modules/custom/openy_font/openy_font.info.yml

```diff
@@ -3,3 +3,4 @@ type: module
 description: OpenY Font Settings.
 core: 8.x
 package: OpenY
```

Comment:
Let's change package to the `OpenY (Deprecated)` And change description - `This module is deprecated and will be removed in future versions. Use FontYourFace module instead.`